components/gitpod/gitpod.libsonnet

```
@@ -198,6 +198,84 @@ function(params) {
     },
   },
 
+  containerdMetricsService: {
+    apiVersion: 'v1',
+    kind: 'Service',
+    metadata: {
+      name: $._config.name + '-containerd-metrics',
+      namespace: $._config.gitpodNamespace,
+      labels: $._config.commonLabels {
+        'app.kubernetes.io/component': 'containerd-metrics',
+      },
+    },
+    spec: {
+      selector: {
+        component: 'containerd-metrics',
+      },
+      ports: [{
+        name: 'metrics',
+        port: 9500,
+      }],
+    },
+  },
+
+  containerdMetricsMonitor: {
```

```suggestion
  containerdMetricsServiceMonitor: {
```

Aha, almost!! Notice that the name of the k8s resource is indeed `ServiceMonitor` and not `Monitor` :)